src/styles/_catalog.scss

```
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
```

In my opinion, it's better to leave `width: 200px` for the card and change the grid 
```suggestion
  grid-template-columns: repeat(1, auto);
```